tabular/src/autogluon/tabular/learner/default_learner.py

```diff
@@ -207,7 +207,7 @@ def general_data_processing(self, X: DataFrame, X_val: DataFrame, X_unlabeled: D
                 else:
                     y_unlabeled = pd.Series(np.nan, index=X_unlabeled.index)
                     y_super = pd.concat([y, y_unlabeled], ignore_index=True)
-                X_super = self.fit_transform_features(X_super, y_super, problem_type=self.label_cleaner.problem_type_transform)
+                X_super = self.fit_transform_features(X_super, y_super, problem_type=self.label_cleaner.problem_type_transform, eval_metric=self.eval_metric)
```

Comment:
why is `eval_metric` needed in this call to `fit_transform_features()`, but not in the call to this same function on L190:  https://github.com/awslabs/autogluon/blob/e70e0e85146bbe11b8bebc2cb782e7415a54ff94/tabular/src/autogluon/tabular/learner/default_learner.py#L190